presto-native-execution/presto_cpp/main/TaskManager.cpp

```diff
@@ -780,6 +784,7 @@ folly::Future<std::unique_ptr<protocol::TaskInfo>> TaskManager::getTaskInfo(
   {
     std::lock_guard<std::mutex> l(prestoTask->mutex);
     prestoTask->updateHeartbeatLocked();
+    prestoTask->updateCoordinatorHeartbeatLocked();
```

Comment:
Can we pass a flag to updateHeartbeatLocked(fromCoordinator)?

Reply:
No, we cannot.
The sites we call these two differ.